com.unity.renderstreaming/Documentation~/en/faq.md

```diff
@@ -4,7 +4,9 @@
 
 ### Video stream not showing in browser
 
-The browser version might be too old. Make sure that the latest version is installed.
+The browser version might be too old. Make sure that the latest version is installed. Or firewall settings can be caused by the issue, please try once to turn off it.
```

Comment:
-> Or firewall settings can be caused by the issue, please try once to turn off it.

Or firewall settings can also cause the issue, please try to turn it off once.